sources/HUBViewControllerImplementation.m

```diff
@@ -48,6 +48,8 @@
 #import "HUBActionHandler.h"
 #import "HUBViewModelDiff.h"
 
+CFTimeInterval const kDownloadingTimeTreshold = 0.07;
```

Comment:
Why does it need to be exposed? Is there any API use case for it? Otherwise I think it's better to keep it internal.